### PR TITLE
Refresh PHP proto types

### PR DESCRIPTION
**Pull request by michaelbausor**
Updates to documentation only

### CODE REVIEW

File: generated/php/google-cloud-error-reporting-v1beta1/proto/src/Google/Devtools/Clouderrorreporting/V1beta1/DeleteEventsRequest.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 /**
  * Deletes all events in the project.
  *
- * Protobuf type <code>Google\Devtools\Clouderrorreporting\V1beta1\DeleteEventsRequest</code>
+ * Generated from protobuf message <code>google.devtools.clouderrorreporting.v1beta1.DeleteEventsRequest</code>
```
Comment (shinfan) [inline generated/php/google-cloud-error-reporting-v1beta1/proto/src/Google/Devtools/Clouderrorreporting/V1beta1/DeleteEventsRequest.php]: Why do you change the package path here? Isn't the path separated by "\" in PHP?

Comment (michaelbausor) [inline generated/php/google-cloud-error-reporting-v1beta1/proto/src/Google/Devtools/Clouderrorreporting/V1beta1/DeleteEventsRequest.php]: Yes, but this is referring to the protobuf message type, which is lowercase and `.` separated. This is a change in PHP protoc which we are now picking up in our generated proto classes.

Comment (shinfan) [inline generated/php/google-cloud-error-reporting-v1beta1/proto/src/Google/Devtools/Clouderrorreporting/V1beta1/DeleteEventsRequest.php]: Okay that make sense

**Comment by shinfan** [comment]
LGTM

**Pull request merged**